### DISCUSSION TITLE
Custom dates to freeze streak

### DISCUSSION THREAD

**Discussion by YorbenVerhoest** [dailydotdev/daily#1185] [Feature Requests]
As I lost my streak during a Belgian holiday, it got me thinking it should be great if we could pre-define some dates where it also auto-freezes. this could also help when going on holiday and I don't have the time or possibility browse.

**Comment by nimrodkra** [comment]
We’re thinking about a solution where you’d be able to restore your last
streak by “paying” with reputation points. Does it make sense?
